static/js/components/search/ModifiedSelectedFilter.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { FILTER_NAMES } from '../../constants';
+import { makeCountryNameTranslations } from '../LearnerSearch';
+export default class ModifiedSelectedFilter extends React.Component {
+  props: {
+    labelKey: string,
+    labelValue: string,
+    removeFilters: Function,
+    bemBlocks?: any,
+    filterId: string
+  };
+
+  countryNameTranslations: Object = makeCountryNameTranslations();
+  render() {
+    const {labelValue, removeFilter, bemBlocks} = this.props;
+    let filterName = this.props.labelKey;
+    let filterValue =
+      (labelValue in this.countryNameTranslations) ? this.countryNameTranslations[labelValue] : labelValue;
+    if (filterName in FILTER_NAMES) {
+      filterName = FILTER_NAMES[filterName];
+    } else if (filterName in this.countryNameTranslations) {
+      filterName = this.countryNameTranslations[filterName];
+    }
+    return <div className={bemBlocks.option()
+    .mix(bemBlocks.container("item"))
+    .mix(`selected-filter--${this.props.filterId}`)()}>
+      <div className={this.props.bemBlocks.option("name")}>{filterName}: {filterValue}</div>
+      <div className={this.props.bemBlocks.option("remove-action")} onClick={removeFilter}>x</div>
+    </div>;
+  }
```

Comment:
I played around with getting rid of the need for custom rendering code, but due to the extensive usage of `this.props` in searchkit I couldn't see a clean way to do it. I did make this `render` method a little more concise in the process, though. What do you think of this?

```javascript
render() {
    let {
      labelKey, 
      labelValue, 
      removeFilter, 
      bemBlocks, 
      filterId
    } = this.props;

    if (labelKey in FILTER_NAMES) {
      labelKey = FILTER_NAMES[labelKey];
    } else if (labelKey in this.countryNameTranslations) {
      labelKey = this.countryNameTranslations[labelKey];
    }
    if (labelValue in this.countryNameTranslations) {
      labelValue = this.countryNameTranslations[labelValue];
    }

    return (
      <div className={
        bemBlocks.option()
        .mix(bemBlocks.container("item"))
        .mix(`selected-filter--${filterId}`)()
      }>
        <div className={bemBlocks.option("name")}>{labelKey}: {labelValue}</div>
        <div className={bemBlocks.option("remove-action")} onClick={removeFilter}>x</div>
      </div>
    );
  }
```